config/environments/development.rb

```diff
@@ -67,4 +67,10 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  # Enable Faraday request header logging
```

Comment:
See comment above.

In development we can manually opt-in to log the headers via the ENV.

Reply:
Done